spotirec/api.py

```diff
@@ -183,24 +187,35 @@ def get_current_artists(self, headers: dict) -> list:
         """
         response = requests.get(f'{self.URL_BASE}/me/player', headers=headers)
         self.error_handle('retrieve current artists', 200, 'GET', response=response)
-        return [str(x['uri'])
-                for x in json.loads(response.content.decode('utf-8'))['item']['artists']]
+        data = json.loads(response.content.decode('utf-8'))
+        try:
+            return [str(x['uri']) for x in data['item']['artists']]
+        except (TypeError, KeyError):
+            return [data['currently_playing_type']]
```

Comment:
Maybe the function docstring should reflect this change, just like the previous.